fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -69,6 +69,18 @@
     @Nonnull
     FDBDatabase getDatabase();
 
+    /**
+     * Get the configuration used in record contexts opened by this runner.
```

Comment:
It's probably worth clarifying that as the object returned is the same object as (not a copy of) the one used by the runner, if one changes one of the values of the builder, then this will affect the values used when the runner creates new contexts.

Reply:
Added.